src/components/ProfilePage/index.jsx

```diff
@@ -5,6 +5,35 @@ import NewsFeed from "../NewsFeed";
 import firebaseWrapper from "../../firebaseWrapper";
 import "./styles.css";
 
+const Curve = ({
+  onClick1,

```

Comment:
I'm not sure what the variable names with `1` and `2` are referring to. Name these better!

Reply:
also im not sure what `activeTabAbove` is either oof